curator-framework/src/main/java/org/apache/curator/framework/imps/CreateBuilderImpl.java

```diff
@@ -485,7 +485,6 @@ private String protectedPathInForeground(String adjustedPath, byte[] data) throw
         }
         catch ( Exception e)
         {
-            ThreadUtils.checkInterrupted(e);
```

Comment:
Most code does not check for `InterruptedException` and this is the problem. Again, please explain what the issue being fixed is here. Is this merely an academic issue or is there a real problem?